cdc/cdc/kv/region_worker.go

```diff
@@ -828,3 +836,17 @@ func RunWorkerPool(ctx context.Context) error {
 	})
 	return errg.Wait()
 }
+
+func GetSafeResolvedTs(resolvedTs uint64) uint64 {
+	cfg := config.GetGlobalServerConfig().KVClient
+
+	logicalTs := oracle.ExtractLogical(resolvedTs)
+	time := oracle.GetTimeFromTS(resolvedTs)
+
+	safeTime := time.Add(-cfg.ResolvedTsSafeInterval)
```

Comment:
If `time` is less than `cfg.ResolvedTsSafeInterval`, suggest to just return the `time`.
A `resolved-ts = 0` would be a legal one ?